src/displayOne.js

```diff
@@ -0,0 +1,19 @@
+import { all } from './index';
```

Comment:
Again avoid import { all }, send it from class to functions, then propagate from function to function, like top to bottom when needed.